fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneRecordCursor.java

```diff
@@ -234,4 +238,8 @@ private void performScan() throws IOException {
         searcher = new IndexSearcher(indexReader);
         topDocs = searcher.search(query, limitRemaining);
```

Comment:
Doesn't `sort` go in here as the third argument? (Maybe only if non-null?)

Reply:
wow.....  🤦‍♀️